plutus-core/plutus-core/src/PlutusCore/Constant/Meaning.hs

```diff
@@ -118,6 +119,15 @@ lookupBuiltin fun (BuiltinsRuntime env) = case env ^? ix fun of
     Nothing  -> throwingWithCause _MachineError (UnknownBuiltin fun) Nothing
     Just bri -> pure bri
 
+-- | Look up the runtime info of a built-in function during evaluation.
+lookupBuiltinExc
+    :: forall ex err fun term m proxy val . (MonadThrow m, ex ~ ErrorWithCause err term, AsMachineError err fun term, Ix fun, Exception ex)
+    => proxy ex -> fun -> BuiltinsRuntime fun val -> m (BuiltinRuntime val)
```

Comment:
Sorry, I understand it's absolutely awful nitpicking, but maybe also rename `ex` to `exc`?